connector/saml/saml_test.go

```diff
@@ -62,17 +62,17 @@ func runVerify(t *testing.T, ca string, resp string, shouldSucceed bool) {
 	}
 }
 
-func newProvider(issuer string, redirectURI string) *provider {
-	if issuer == "" {
-		issuer = defaultIssuer
+func newProvider(ssoIssuer string, redirectURI string) *provider {
+	if ssoIssuer == "" {
+		ssoIssuer = defaultSSOIssuer
 	}
 	if redirectURI == "" {
 		redirectURI = defaultRedirectURI
 	}
 	now, _ := time.Parse(time.RFC3339, "2017-01-24T20:48:41Z")
 	timeFunc := func() time.Time { return now }
 	return &provider{
-		issuer:       issuer,
```

Comment:
Sorry, trying to re-review this. Issuer is no longer being used here then, correct? I was under the impression that `issuer` was supposed to be the value that you've changed to `ssoIssuer` to anyway. E.g. that `issuer` should just be `http://www.okta.com/exk91cb99lKkKSYoy0h7`.

Maybe this an issue with the documentation?